src/systemctl/systemctl-enable.c

```diff
@@ -245,6 +245,10 @@ int verb_enable(int argc, char *argv[], void *userdata) {
                         if (r < 0)
                                 return r;
                 }
+
+                if (streq(verb, "mask") && !arg_quiet && !arg_no_warn)
```

Comment:
why do this only in case of masking? sounds worthy to warn in case of disabling too I think

Reply:
instead of checking the `verb` here again I think it would be nice to instead add a local variable here called `warn_trigger` or so that is set in the if/else blocks above in the cases where we want this warning to happen, i.e. for mask + disable at least. 

could even be named `warn_trigger_message` or so and be of type `const char*`, where you just put the string `"Masking"` or `"Disabling` in that you pass as third arg to `warn_triggering_units()`